### ISSUE TITLE
[BUG] Scheduled Job Fails on Windows 10

### ISSUE THREAD

**Issue by YoloWingPixie** [YoloWingPixie/lsobot#13]
**Describe the bug**
When running the bot on Windows 10 and following the installation instructions, LsoBot-Register will run and say the job is created, but the bot will not write any debug logs and traps are not recorded.

Checking in Task Scheduler, Task Scheduler Library -> Microsoft -> Windows -> Powershell -> ScheduledJobs -> LSO Check will show this error:

`Task Scheduler failed to start "\Microsoft\Windows\PowerShell\ScheduledJobs\LSO Check" task for user "REDACTED". Additional Data: Error Value: 2147943711.`

**Environment Info:**
 - DCS Server OS: Windows 10
 - DCS Verision: 2.7.0
 - Highest version of Powershell installed: 5.1
 - LsoBot Version: 1.1.0

**Comment by YoloWingPixie** [comment]
Fix for now:

Open Task Scheduler, you can search for it at your Start Menu

Navigate to Task Scheduler Library -> Microsoft -> Windows -> Powershell -> ScheduledJobs

Right Click on `LSO Check` and hit `Properties`

Towards the bottom of the `General` tab there hsould be a `Security Options` section with your username filled in. Uncheck `Do not store password.`.

Hit ok.

Enter your password when prompted (It's encrypted still and only Windows sees this, not the script).

Wait up to 10 minutes or right-click the job again and hit `Run Now`

If you don't feel comfortable with the script running with the permissions of your user account (because it's an admin or something like that), you can make a new account and lock it down to only have access to the LsoBot folder and your account's Saved Games\DCS folder.

Basically, Windows 10 is slightly more locked down than Windows Server when it comes to Powershell and doesn't give Scheduled Jobs access to as many privileges by default.

Continuing to investigate if there's a way around this or if the installation instructions for Windows 10 just need to include this step.

**Comment by ags313** [comment]
I had problems on win10. I scheduled job by executing register from an elevated powershell process, AND making path call explicit, for some reason that didn't work :/

**Comment by YoloWingPixie** [comment]
> I had problems on win10. I scheduled job by executing register from an elevated powershell process, AND making path call explicit, for some reason that didn't work :/

I had realized after posting the above that on my system I had my execution policy set to bypass. Try doing that, as having the ExecutionPolicy in Unrestricted is not sufficient for a Scheduled Job on Windows 10 since Windows 10 still tries to pester the session to allow it:

`Set-ExecutionPolicy -ExecutionPolicy Bypass`
